packages/client/src/views/PrintCertificate/Payment.tsx

```diff
@@ -207,7 +208,7 @@ function mapStatetoProps(
   ) as IPrintableDeclaration | undefined
 
   if (!declaration) {
-    throw new Error(`Declaration "${registrationId}" missing!`)
+    return <Redirect to={HOME} />
```

Comment:
@favour234 the `<Redirect />` needs to be called from inside a component and not from the `mapStateToProps` function

Reply:
okay..was actually trying to test something on the error i was encountering..would be refactored back to the error message.